scripts/Diffraction/isis_powder/polaris_routines/polaris_algs.py

```diff
@@ -132,11 +132,13 @@ def generate_ts_pdf(run_number, focus_file_path, merge_banks=False, q_lims=None,
     mantid.RenameWorkspace(InputWorkspace='focused_ws', OutputWorkspace=run_number+'_focused_Q')
     if isinstance(focused_ws, WorkspaceGroup):
         for i in range(len(focused_ws)):
-            mantid.RenameWorkspace(InputWorkspace=focused_ws[i], OutputWorkspace=run_number+'_focused_Q_'+str(i+1))
+            if str(focused_ws[i]) != (run_number+'_focused_Q_'+str(i+1)):
```

Comment:
This is a useful extra check. Can you set up the target file name in a string variable first and then just use that string variable on lines 135 and 136? Will make it easier to update the required file name format in future
Similar comment on lines 140\141